docker-secrets.yml

```
@@ -0,0 +1,101 @@
+services:
+  mdv_app:
+    build:
+     context: .
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)